src/main/java/org/isf/admission/service/AdmissionIoOperations.java

```diff
@@ -264,7 +259,7 @@ public int getNextYProg(String wardId) throws OHServiceException {
 	public static boolean afterJune;
 
 	public static LocalDateTime getNow() {
-		LocalDateTime now = TimeTools.getNow();
+		LocalDateTime now = LocalDateTime.now();
```

Comment:
Please do not change this; the method `getNow()` is used so nanoseconds are not included in the time and thus can be used successfully across operating systems and architectures without issue.

Reply:
I made the rebase incorrectly